README.md

```diff
@@ -89,6 +89,7 @@ complexify his philosophy… (I'm pretty sure, i now did complexify it :D) ^^
 Please see default value by Operating System file in [vars][vars directory] directory.
 
 * **nft_pkg_list** : The list of package(s) to provide `nftables`.
+* **nft_bin_location** : Path to `nftbles` executable. [default : `/usr/bin/nft`]
```

Comment:
Typo in `nftbles`

Reply:
Fixed along with typo in executable path (affected only README)